### PR TITLE
layers: Fix rasterizationOrderAttachmentAccess

### PR DESCRIPTION
**Pull request by spencer-lunarg**
Fixes VUs for `VK_EXT_rasterization_order_attachment_access` to check if `VkRenderPass` is null or not (now that there is `VK_KHR_dynamic_rendering_local_read`)

Also some VUID label churn

Lastly, tried moving some stuff out of the monolithic  `manual_PreCallValidateCreateGraphicsPipelines`

### CODE REVIEW

File: layers/stateless/sl_pipeline.cpp [hunk 1]
```diff
@@ -909,62 +978,49 @@ bool StatelessValidation::manual_PreCallValidateCreateGraphicsPipelines(
...
-                                     "rasterizationOrderStencilAttachmentAccess == VK_FALSE, but "
-                                     "VkPipelineDepthStencilStateCreateInfo::flags == %s",
-                                     string_VkPipelineDepthStencilStateCreateFlags(create_info.pDepthStencilState->flags).c_str());
+                if ((ds_state.flags &
+                     VK_PIPELINE_DEPTH_STENCIL_STATE_CREATE_RASTERIZATION_ORDER_ATTACHMENT_STENCIL_ACCESS_BIT_EXT) != 0) {
+                    if (!rasterization_order_attachment_access_feature ||
+                        !rasterization_order_attachment_access_feature->rasterizationOrderStencilAttachmentAccess) {
+                        skip |= LogError(
+                            "VUID-VkPipelineDepthStencilStateCreateInfo-rasterizationOrderStencilAttachmentAccess-06464", device,
+                            ds_loc.dot(Field::flags), "(%s) but rasterizationOrderDepthAttachmentAccess feature is not enabled",
```
Comment (janharaldfredriksen-arm) [inline layers/stateless/sl_pipeline.cpp:1014]:
```suggestion
                            ds_loc.dot(Field::flags), "(%s) but rasterizationOrderStencilAttachmentAccess feature is not enabled",
```

File: layers/stateless/sl_pipeline.cpp [hunk 2]
```diff
@@ -976,27 +1032,20 @@ bool StatelessValidation::manual_PreCallValidateCreateGraphicsPipelines(
...
+                     VK_PIPELINE_COLOR_BLEND_STATE_CREATE_RASTERIZATION_ORDER_ATTACHMENT_ACCESS_BIT_EXT) != 0) {
+                    if (!rasterization_order_attachment_access_feature ||
+                        !rasterization_order_attachment_access_feature->rasterizationOrderColorAttachmentAccess) {
                         skip |= LogError("VUID-VkPipelineColorBlendStateCreateInfo-rasterizationOrderColorAttachmentAccess-06465",
-                                         device, color_loc,
-                                         "VkPhysicalDeviceRasterizationOrderAttachmentAccessFeaturesARM::"
-                                         "rasterizationColorAttachmentAccess == VK_FALSE, but "
-                                         "VkPipelineColorBlendStateCreateInfo::flags == %s",
+                                         device, color_loc.dot(Field::flags),
+                                         "(%s) but rasterizationOrderDepthAttachmentAccess feature is not enabled",
```
Comment (janharaldfredriksen-arm) [inline layers/stateless/sl_pipeline.cpp:1040]:
```suggestion
                                         "(%s) but rasterizationOrderColorAttachmentAccess feature is not enabled",
```

**Review by janharaldfredriksen-arm — COMMENTED** [review]
Nice to have these ARM->EXT renames done. (I meant to, but never got around to it.)

File: layers/stateless/sl_pipeline.cpp (hunk 1 above)
Comment (spencer-lunarg) [reply layers/stateless/sl_pipeline.cpp:1014]: o man, these 3 always get my eyes crossed... thanks for the extra eyes

**Pull request merged**